### PR TITLE
Portfolio: Complete Mobile Version

### PR DESCRIPTION
**Pull request by fdezcaminero**
To the previous project where I started building my portfolio, I have added:

* Work Section: Containing projects I have worked on
* About Section: Listing my skills in programming and frameworks

It is designed after a Figma model, and makes use of Flexbox, CSS Grid, among other new tricks.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,5 +26,183 @@ <h1>José Fernández-Caminero</h1>
             <img src="resources/image_geometry_2.svg" alt="Geometrical decoration 2">
```
Comment (mgmediaweb) [inline index.html]: Kindly, decoration images should be part of the background of your site, I recommend using background-image

File: index.html [hunk 2]
```diff
@@ -26,5 +26,183 @@ <h1>José Fernández-Caminero</h1>
...
+
+        <button>Get my CV
+            <img src="resources/ic_download.svg" alt="Downward Arrow">
+        </button>
+
+        <section class="marginleft">
+            <section class="rectangle">
+                <img src="resources/Rectangle 41.png" alt="Blue Rectangle">
+
+                <img class="rectanglesymbol1" src="resources/pound.svg" alt="Pound Sign">
```
Comment (mgmediaweb) [inline index.html]: Kindly, decoration images should be part of the background of your site, I recommend using background-image

File: index.html [hunk 3]
```diff
@@ -26,5 +26,183 @@ <h1>José Fernández-Caminero</h1>
...
+                        
+                        <li class="maryPoppins2">CSS</li>
+                    </ul>
+                </section>
+            </section>
+
+            <section class="rectangle">
+                <img src="resources/Rectangle 39.png" alt="Red Rectangle">
+
+                <img class="rectanglesymbol2" src="resources/Group 18.png" alt="Square inside a Square">
```
Comment (mgmediaweb) [inline index.html]: Kindly, decoration images should be part of the background of your site, I recommend using background-image

File: index.html [hunk 4]
```diff
@@ -26,5 +26,183 @@ <h1>José Fernández-Caminero</h1>
...
+                        
+                        <li class="maryPoppins2">Selenium</li>
+                    </ul>
+                </section>
+            </section>
+
+            <section class="rectangle">
+                <img src="resources/Rectangle 42.png" alt="Yellow Rectangle">
+
+                <img class="rectanglesymbol3" src="resources/ic_skills.svg" alt="Lit Lightbulb">
```
Comment (mgmediaweb) [inline index.html]: Kindly, decoration images should be part of the background of your site, I recommend using background-image

File: index.html [hunk 5]
```diff
@@ -26,5 +26,183 @@ <h1>José Fernández-Caminero</h1>
...
+                        
+                        <li class="maryPoppins2">Capibara</li>
+                        
+                        <li class="maryPoppins2">Selenium</li>
+                    </ul>
+                </section>
+            </section>
+
+            <section class="rectangle">
+                <img src="resources/Rectangle 42.png" alt="Yellow Rectangle">
```
Comment (mgmediaweb) [inline index.html]: Kindly, I recommend that you change these background color images and use background colors with CSS.

File: index.html [hunk 6]
```diff
@@ -26,5 +26,183 @@ <h1>José Fernández-Caminero</h1>
...
+                        
+                        <li class="maryPoppins2">HTML</li>
+                        
+                        <li class="maryPoppins2">CSS</li>
+                    </ul>
+                </section>
+            </section>
+
+            <section class="rectangle">
+                <img src="resources/Rectangle 39.png" alt="Red Rectangle">
```
Comment (mgmediaweb) [inline index.html]: Kindly, I recommend that you change these background color images and use background colors with CSS.

File: index.html [hunk 7]
```diff
@@ -26,5 +26,183 @@ <h1>José Fernández-Caminero</h1>
...
+            I love to code. I think coding is where the arts and the sciences meet. It's the most natural activity for a renaissance person.
+        </p>
+
+        <button>Get my CV
+            <img src="resources/ic_download.svg" alt="Downward Arrow">
+        </button>
+
+        <section class="marginleft">
+            <section class="rectangle">
+                <img src="resources/Rectangle 41.png" alt="Blue Rectangle">
```
Comment (mgmediaweb) [inline index.html]: Kindly, I recommend that you change these background color images and use background colors with CSS.

File: index.html [hunk 8]
```diff
@@ -26,5 +26,183 @@ <h1>José Fernández-Caminero</h1>
...
+                        <li class="maryPoppins2">Bootstrap</li>
+                        
+                        <li class="maryPoppins2">Ruby on rails</li>
+                        
+                        <li class="maryPoppins2">RSpec</li>
+                        
+                        <li class="maryPoppins2">Capibara</li>
+                        
+                        <li class="maryPoppins2">Selenium</li>
+                    </ul>
```
Comment (mgmediaweb) [inline index.html]: Kindly, I recommend you remove those blank lines, so that your code looks more orderly

File: index.html [hunk 9]
```diff
@@ -26,5 +26,183 @@ <h1>José Fernández-Caminero</h1>
...
+                    <ul>
+                        <li class="maryPoppins2">Codekit</li>
+                        
+                        <li class="maryPoppins2">Codepen</li>
+                        
+                        <li class="maryPoppins2">GitHub</li>
+                        
+                        <li class="maryPoppins2">GitLab</li>
+                        
+                        <li class="maryPoppins2">Terminal</li>
```
Comment (mgmediaweb) [inline index.html]: Kindly, I recommend you remove those blank lines, so that your code looks more orderly

File: index.html (hunk 1 above)
Comment (mgmediaweb) [reply index.html]: Kindly, by convention the FOOTER semantic tag is used in the last part of your website, in your case I recommend using DIV

File: index.html [hunk 10]
```diff
@@ -26,5 +26,183 @@ <h1>José Fernández-Caminero</h1>
             <img src="resources/image_geometry_2.svg" alt="Geometrical decoration 2">
         </footer>
     </section>
+
+    <section>
+        <h1>Projects</h1>
+
+        <section class="box1">
```
Comment (mgmediaweb) [inline index.html]: Kindly, I recommend you to use SECTION tags for each theme of your page, inside these, you could only use DIV, H, P, etc, but using SECTION inside SECTION is not well applied.

File: index.html [hunk 11]
```diff
@@ -26,5 +26,183 @@ <h1>José Fernández-Caminero</h1>
...
+            </ul>
+
+            <button class="bordersolid">See this project
+                <img src="resources/Union.png" alt="Cool Arrow Pointing Right">
+            </button>
+        </section>
+    </section>
+
+    <section>
+        <h1>About</h1>
```
Comment (mgmediaweb) [inline index.html]: Kindly, by the accessibility rule there can only be one H1 tag on your page, the others follow in order of importance, H2, H3, etc.

File: css/styles.css [hunk 12]
```diff
@@ -54,3 +54,83 @@ button:active {
   background: white;
   border-width: 2px 2px 2px 2px;
 }
+
+.box1 {
+  background: #1a2236;
+  display: grid;
+  justify-items: center;
+  width: 320px;
```
Comment (mgmediaweb) [inline css/styles.css]: Kindly, your site should adapt to any size of mobile device, I recommend you not to limit your window with static measurements but use width 100%

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

work6


## Highlights 💯 

- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️

- I kindly recommend you to better follow the guidelines of the original Figma design.

image
image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- I kindly recommend you remove the elements that your site does not need (in your case some images from the About section)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 11 above)
Comment (fdezcaminero) [reply index.html]: @mgmediaweb I will make the changes, no prob. But are you sure about this? Could you pass some guidelines that point this out? I've looked this up and I find that you can use more than one h1, just not nested within themselves. I don't mind complying, but I would love to understand the matter a little bit better. Please send guidelines that defend the case. Thanks.

File: css/styles.css (hunk 12 above)
Comment (fdezcaminero) [reply css/styles.css]: @mgmediaweb  Hi Gonzalo. I've done it the way you asked. But I thought this was not supposed to be responsive yet. After a certain screen size the design should change. If I change the size of the pictures with your approach it doesn't look good (I think). But I hope it complies with your requests.

File: index.html (hunk 11 above)
Comment (Ismailco) [reply index.html]: @mgmediaweb, for SEO reasons you should always use only one `<h1>` tag on the page, that represent a title or the most important header in that page

**Review by Ismailco — APPROVED** [review]
### Hi @fdezcaminero, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**